manim/utils/ipython_magic.py

```diff
@@ -168,7 +168,7 @@ def construct(self):
 
                 file_type = mimetypes.guess_type(config["output_file"])[0]
                 embed = config["media_embed"]
-                if embed is None:
+                if embed is False:
```

Comment:
What about this?
```suggestion
                if not embed:
```
It's more Pythonic. According to PEP8:
>> Don’t compare boolean values to True or False using `==`